src/main/java/com/minecolonies/coremod/blocks/BlockHutStonemason.java

```diff
@@ -1,7 +1,18 @@
 package com.minecolonies.coremod.blocks;
 
-import net.minecraft.block.Block;
+import com.minecolonies.api.colony.permissions.Action;
+import com.minecolonies.coremod.client.gui.WindowGuiCrafting;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.buildings.views.AbstractBuildingView;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.Minecraft'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)